conf/provisioning/alerting/sample.yaml

```diff
@@ -0,0 +1,165 @@
+# # config file version
+apiVersion: 1
+
+# # List of rule groups to import or update
+# groups:
+#     # <int> organization id, default = 1
```

Comment:
```suggestion
#     # <int> organization ID, default = 1
```

Reply:
Or identifier, you use "id", "ID", and "identifier" in this file and this needs to be standardized.

Reply:
Changed everywhere 👍